vibe_code_apps_22/eplus_gym_app/site_config.py

```
@@ -0,0 +1,645 @@
+        )
+        st.caption(
+            f"Occupied deadband (derived): **{float(occ_cool) - float(occ_heat):.1f} F** "
+            "(occ cool - occ heat)"
+        )
+
+        st.markdown("**Weekly people + HVAC**")
+        opt_on = bool(cfg.get("optimum_start", False))
+        st.caption(
+            "People drives occupancy and plug loads. HVAC is equipment availability "
+            + (
+                "(greyed: **optimum start** drives HVAC start from people start − lead; "
```

Ruff (0.16.1)

[warning] String contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)